docs/installation/migrate.rst

```diff
@@ -114,24 +135,21 @@ To migrate |sr| to |ccloud|, follow these steps:
     
 #.  Stop all consumers.
 
-    .. code:: bash
-
-        TBD example output 
 
 #.  Configure all consumers to point to the destination |sr| in the cloud and restart them.
 
-    .. code:: bash
-
-        TBD example output 
+    For example, if you are configuring |sr| in a Java client, change |sr| URL
+    from source to destination either in the code or in a properties file that
+    specifies the |sr| URL, type of authentication USER_INFO, and credentials).
+    
+    See :ref: schema_registry_tutorial.rst#java-consumers for further examples.
+    
 
 #.  Configure all producers to point to the destination |sr| in the cloud and restart them.
```

Comment:
Should producers be combined with the step above?

Reply:
@ybyzek I recommend that we should separate consumers and producers steps explicitly.